modules/virt-creating-vm-web-page-cli.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * virt/creating_vms_advanced/creating_vms_advanced_web/virt-creating-vms-from-web-images.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-creating-vm-import-cli_{context}"]
```

Comment:
This ID should probably be updated to match the file name

Reply:
I initially did but this module is referenced multiple times and possibly linked form external sources. This was the least invasive way, but if you insist, I can change it. The cherry-picks will be a nightmare.

Reply:
I don't insist :)

Reply:
Thank you! And thank you very much for your thorough review, I am working on fixing the other parts and will retest the commands tomorrow to get expected output for verification steps.